_includes/topNavImageLinks.html

```diff
@@ -0,0 +1,8 @@
+<div class="imageLinks">
+    <button class="menuToggle" onLoad="handleScroll" onClick="toggleMenu()">
+        <img width="20" height="20" src="/assets/img/monochrome/menu.svg" alt="Open menu">
+    </button>
+    <a class="logo" href=".">
+        <img width="60" height="60" src="/assets/img/monochrome/logo-with-wordmark.svg" alt="openSUSE logo">
```

Comment:
I think we should do a cobranding of openSUSE and `Get`, to make sure that people understand that this link leads to this website and not to openSUSE.org. You should also not use `.` as the url, since that doesn't make much sense

Reply:
I agree that the site can do a better job of clarifying the hierarchy between opensuse.org and get.opensuse.org. However, the "Get" title on get.opensuse.org confused me the first time I saw it, so perhaps I could find another way of clarifying the difference.

Could we change the "Get" to something that better clarifies the purpose of the site? We could write something like "openSUSE OS" or "openSUSE distro" to clarify that get.opensuse.org is about the Linux product (as opposed to the build service for example). Perhaps I could add some kind of "up one directory"-style arrow to the left of the logo to clarify that this is how you return to opensuse.org?

Reply:
I played with the branding some more more. Replacing 'Get' with 'Linux' feels like a clear way to communicate the purpose of get.opensuse.org. It's the website with all openSUSE Linux flavors.

![image](https://github.com/user-attachments/assets/71d403dd-30f9-4d5f-b5e3-45240901ae61)

![image](https://github.com/user-attachments/assets/62a2d5c9-a3ba-4ee4-bc8d-74131076b767)